osu.Game/Screens/Play/HUD/SkinnableInfo.cs

```diff
@@ -32,6 +32,13 @@ public class SkinnableInfo : IJsonSerializable
 
         public Anchor Origin { get; set; }
 
+        /// <summary>
+        /// <see langword="false"/> if this <see cref="ISkinnableDrawable"/>'s <see cref="Drawable.Anchor"/> is
+        /// automatically determined by proximity, <see langword="true"/> if the user has overridden it.
+        /// </summary>
+        /// <remarks>Stored this way because <c>default(bool)</c> is <see langword="false"/> and we want the default behaviour to be "closest".</remarks>
+        public bool IsNotUsingClosestAnchor { get; set; }
```

Comment:
```suggestion
        public bool IsUsingClosestAnchor { get; set; }
```

Generally, it is good practice to use positive term for boolean property/variable. Also I see so many negation when using this property in this PR, so just change it to avoid that.

Reply:
I just wanted it to be very clear about what it's storing. Would something like `OverrideAnchor` be better?

As I mention in the doc comment, it must be stored in a way that `false` means "use closest."

Reply:
Could append `= true` to the line in that case? also regarding name, I would shorten it to `UsingClosestAnchor`.

Reply:
> Could append `= true` to the line in that case? also regarding name, I would shorten it to `UsingClosestAnchor`.

I'm not sure if that works with the serialization, but that would be preferable from a code quality perspective. I'll test it out.